tests/drivers/test_raster_drivers.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy as np
 
-DRIVERS = ['sqlite', 'mysql']
+DRIVERS = ['sqlite', 'mysql', 'postgresql']
```

Comment:
I don't really see the point, since the meta storage is now completely decoupled from the raster retrieval. I would actually suggest this instead:

```suggestion
DRIVERS = ['sqlite']
```

Reply:
Yeah I agree, as long as the postgres metastore interface works as expected then it should not have any impact on the raster driver. We are doing unit tests after all.